controller/lib/debug/debug.cpp

```diff
@@ -272,30 +272,36 @@ void DebugSerial::ProcessCmd() {
   // The length that we pass in to the command handler doesn't
   // include the command code or CRC.  The max size is also reduced
   // by 3 to make sure we can add the error code and CRC.
-  int len = buffNdx - 3;
-  DbgErrCode err =
-      cmd->HandleCmd(&cmdBuff[1], &len, static_cast<int>(sizeof(cmdBuff) - 3));
+  int len_in = buffNdx - 3;
```

Comment:
This isn't your code, but the crc itself is 4 bytes long, right?  I dunno how long the error code is, but isn't this already too short?

Reply:
I think we're using a 16-bit CRC at the moment

Reply:
Yeah it's a custom 16-bit implementation. I considered taking a stab at replacing it with the 32-bit one so we can use our existing checksum.h on C++ side and zlib.crc32 on Python side, but it seemed like it'd be much easier to do after using the framing library and protobufs.